db/schema.rb

```diff
@@ -24,4 +31,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
This was dragged from another branch it is not part of this PR.
rails db:drop + rails db:create + delete schema file + rails:db migrate + rails db:seed

Reply:
Done